docs/api/platform/flow_control.md

```diff
@@ -2,51 +2,68 @@
 
 We can use Blinky to explore flow control and task management in Arm Mbed OS applications. We'll look at automated actions first, then move on to handling user actions.
 
-## Flow control for automated actions
+## Flow control for delayed actions
 
-If we want to automatically blink an LED, we have three main techniques:
+If we want to automatically blink an LED, we have four main techniques:
 
-1. [Wait](#wait)
-1. [Ticker](#ticker)
+1. [Busy wait](#busy-wait)
+1. [Ticker, Timeout](#ticker-timeout)
 1. [Thread](#thread)
+1. [EventQueue](#eventqueue)
 
-### Wait
+All above techniques accomplish delays but cater to different requirements for precision, efficiency and context.
 
-#### Busy wait
+<span class="tips">**Tip:** You may want to read the [power optimization](../apis/platform-concepts.html) tutorial to understand how to achieve power savings. </span>
```

Comment:
```suggestion
<span class="tips">**Tip:** You may want to read the [power optimization](../apis/platform-concepts.html) tutorial to learn how to save power. </span>
```

Reply:
Is this related specifically to automated/delayed actions, or should it be before this section (as a generic "flow control is related to power optimization and you should read both")?

Reply:
I think "Further readings" would be a better place. It's not a strict prerequisite, and this tutorial discusses a bit around power optimization along the way. Putting the link at the end minimises distraction in my opinion.